### PR TITLE
Add sentry error reporting

### PR DESCRIPTION
**Pull request by eliecharra**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | no
| 🚀 New feature?   | yes
| ⚠ Deprecations?   | no
| ❌ BC Break       | no
| 🔗 Related issues | #58 
| ❓ Documentation  | no

## Description

Add crash reporting flag

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -46,4 +70,12 @@ func main() {
...
 		}
 	}
+
+	return 0
+}
+
+func flushSentry() {
+	fmt.Print("Sending error report ...")
+	sentry.Flush(60 * time.Second)
+	fmt.Printf(" done, thanks you %s\n", color.RedString("❤️"))
```
Comment (wbeuil) [inline main.go:80]: Should be ` done, thank you %s\n` or ` done, thanks %s\n`

Comment (eliecharra) [reply main.go]: Fixed

File: pkg/cmd/driftctl.go [hunk 2]
```diff
@@ -54,6 +61,7 @@ func NewDriftctlCmd(build build.BuildInterface) *DriftctlCmd {
 
 	cmd.PersistentFlags().BoolP("help", "h", false, "Display help for command")
 	cmd.PersistentFlags().BoolP("no-version-check", "", false, "Disable the version check")
+	cmd.PersistentFlags().BoolP("error-reporting", "", false, "Enable error reporting.\nWARNING: may leak some sensitive data")
```
Comment (wbeuil) [inline pkg/cmd/driftctl.go:64]: Should be `may leak sensitive data` since data is invariable

Comment (eliecharra) [reply pkg/cmd/driftctl.go]: Fixed

**Review by wbeuil — APPROVED** [review]
LGTM

**Pull request merged**